### ISSUE TITLE
Add ChipView for web search interface

### ISSUE THREAD

**Issue by koppor** [JabRef/jabref#7806]
The issue is detailed at https://github.com/koppor/jabref/issues/341. Please follow that link.

**Comment by betochf** [comment]
Does someone still working on this, I would like to try it

**Comment by koppor** [comment]
@betochf Just go ahead. I would propose to work on https://github.com/koppor/jabref/issues/341. Then, one can work on https://github.com/JabRef/jabref/issues/7423. WDYT?

**Comment by betochf** [comment]
@koppor, yeah that's great... will give them a look and see which one I can make
